### PR TITLE
regex updated rfc3164

### PR DESCRIPTION
**Pull request by nandinivij**
Resolve Ingestion failure of Github TA Syslog event - [Ingestion failure](https://github.com/splunk/splunk-add-on-for-github-audit-log-monitoring/runs/4471039458)

### CODE REVIEW

**Review by samirhadfa-crest — APPROVED** [review]

File: tests/unit/tests_standard_lib/test_requirement_tests/test_test_generator.py [hunk 1]
```diff
@@ -106,7 +106,7 @@ def test_extract_params():
             ["requirement.log"],
             [True],
             ["syslog"],
-            {"event": ["<34>Oct 11 22:14:15 machine1 event_1"]},
+            {"event": ["<34>Oct 11 22:14:15 machine1 pr1:event_1"]},
```
Comment (arys-splunk) [inline tests/unit/tests_standard_lib/test_requirement_tests/test_test_generator.py:109]: Basically, our test sample was incorrect, that's why we did not catch this issue?

Comment (nandinivij) [reply tests/unit/tests_standard_lib/test_requirement_tests/test_test_generator.py:109]: yup true

**Review by arys-splunk — APPROVED** [review]

**Pull request merged**